panel/src/components/Forms/Input/SelectInput.vue

```diff
@@ -157,6 +163,10 @@ export default {
 	padding: var(--input-padding);
 	border-radius: var(--input-rounded);
 }
+.k-select-input[data-empty="true"] {
```

Comment:
I'm using the same gray color here as the other text inputs use for their placeholder.